src/Motor.Extensions.Utilities/WrapperMotorHost.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+using Sentry;
+
+namespace Motor.Extensions.Utilities;
+
+internal class WrapperMotorHost : IHost
+{
+    private readonly IHost _host;
+
+    public WrapperMotorHost(IHost host)
+    {
+        _host = host;
+    }
+
+    public void Dispose()
+    {
+        _host.Dispose();
+    }
+
+    public async Task StartAsync(CancellationToken cancellationToken = default)
+    {
+        try
+        {
+            await _host.StartAsync(cancellationToken);
+        }
+        catch (Exception e)
+        {
+            if (SentrySdk.IsEnabled)
```

Comment:
is this required? I thought `UseSentry` would also register a global exception handler?